diff --git a/node_modules/react-native-mlkit-ocr/android/build.gradle b/node_modules/react-native-mlkit-ocr/android/build.gradle
index c2bcb60..e7ffb24 100644
--- a/node_modules/react-native-mlkit-ocr/android/build.gradle
+++ b/node_modules/react-native-mlkit-ocr/android/build.gradle
@@ -4,11 +4,11 @@ buildscript {
 
   repositories {
     google()
-    jcenter()
+    mavenCentral()
   }
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.2.1'
+    classpath 'com.android.tools.build:gradle:8.0.2'
     // noinspection DifferentKotlinGradleVersion
     classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
   }
@@ -52,7 +52,6 @@ android {
 
 repositories {
   mavenCentral()
-  jcenter()
   google()
 
   def found = false
